dialoptions.go

```diff
@@ -232,13 +232,6 @@ func WithBalancerName(balancerName string) DialOption {
 	})
 }
 
-// withResolverBuilder is only for grpclb.
-func withResolverBuilder(b resolver.Builder) DialOption {
-	return newFuncDialOption(func(o *dialOptions) {
-		o.resolverBuilder = b
```

Comment:
Move this fields to ClientConn, now it's not part of the config?

Reply:
Done.